python_modules/libraries/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing Libraries
+Welcome! We appreciate your interest in contributing to Dagster Integrations.
+This guide outlines the steps for community members who wish to contribute to
+integratiosn to the project.
+
+## Dagster Libraries
+Dagster Libraries are integrations that can be installed separately from the
+main Dagster package, and provide additional functionality with external tools,
+services, and APIs.
+
+Dagster Libraries may be maintained by either the **Dagster Team** or by
+**Dagster Community** members. Dagster libriaries will include a
+`integration.yaml` in each subfolder, which define, among other things,
+the maintainer of each package.
+
+## integration.yaml
+The `integration.yaml` file must include the following:
+
+```
+package-name: The name of the python package, for example, `dagster-duckdb`
+name: The name of the library, which may be different than the package name
```

Comment:
Would it make sense to pull this from the setup.py?

Reply:
My only concern was if we decided to use these integrations.yaml files to build our integrations page with metadata. Given that contributors may elect to use setup.py, setup.cfg, or pyproject.toml to build a package I wanted to make it easier for us to scan and parse.

The other option would be to be more prescriptive on how packages are defined, but I'm fine either way

Reply:
I think we should be prescriptive about how packages are defined, because it will make it easier for us to manage.